lib/authHandlers.js

```diff
@@ -8,6 +8,8 @@ var sessions       = require('client-sessions'),
     config         = require('./config'),
     parseDuration  = require('parse-duration');
 
+var secondarySessionTokens = {}
```

Comment:
Shouldn't we be concerned with the cache growing without bounds?  Do we need to purge periodically?